app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -67,18 +71,18 @@
     private static final String NEWLINE = "\n";
     private static final Region region = Region.getRegion(Regions.US_EAST_1);
 
-    private String supportEmail;
+    private String supportEmail = "support@sagebridge.org";
```

Comment:
This should work. The string is just a bean. It looks like GitHub stripped out my XML example, but see numAsyncWorkerThreads in application-context.xml for an example.